src/core/src/local_loggers/CompositeLogger.py

```diff
@@ -34,9 +38,17 @@ def __init__(self, file_logger=None, current_env=None):
     @staticmethod
     def log(message):
         """log output"""
-        message = CompositeLogger.__remove_substring_from_message(message, Constants.ERROR_ADDED_TO_STATUS)
-        for line in message.splitlines():  # allows the extended file logger to strip unnecessary white space
-            print(line)
+        try:
+            message = CompositeLogger.__remove_substring_from_message(message, Constants.ERROR_ADDED_TO_STATUS)
+            for line in message.splitlines():  # allows the extended file logger to strip unnecessary white space
+                sys.stdout.write(line)
+                sys.stdout.flush()
+        except IOError as e:
+            #ToDo: Error Handling??
+            if e.errno == errno.EPIPE:
+                return
```

Comment:
Since we cannot continue execution without log output working (to avoid blind execution that cannot be inspected later), a viable solution must be implemented to correct the issue. If the attempt to correct it fails (with retries, if retries are predicted to help in any way), execution must still stop.